source/glest_map_editor/main.cpp

```diff
@@ -334,17 +333,12 @@ void MainWindow::init(string fname) {
 	toolbar->AddTool(miBrushResource + 5, _("resource4"), wxBitmap(brush_resource_4), _("custom4"));
 	toolbar->AddTool(miBrushResource + 6, _("resource5"), wxBitmap(brush_resource_5), _("custom5"));
 	toolbar->AddSeparator();
-	toolbar->AddTool(miBrushObject + 1, _("brush_none"), wxBitmap(brush_none), _("None (erase)"));
```

Comment:
@titiger , you see in the following 11 lines, within the quotes (scroll to the right), the object name, and that text is all duplicated (with some differences) in the header file. What my PR does is merge it.